src/schemas/sql/ibtracks_tracks_geo.sql

```diff
@@ -0,0 +1,37 @@
+-- Table: storms.ibtracs_tracks_geo
+
+-- DROP TABLE IF EXISTS storms.ibtracs_tracks_geo;
+
+CREATE TABLE IF NOT EXISTS storms.ibtracs_tracks_geo(
+    wind_speed INTEGER CHECK (wind_speed BETWEEN -1 AND 300),
+    pressure INTEGER CHECK (pressure BETWEEN 800 AND 1100),
+    max_wind_radius INTEGER CHECK (max_wind_radius >= 0),
+    last_closed_isobar_radius INTEGER CHECK (last_closed_isobar_radius >= 0),
+    last_closed_isobar_pressure INTEGER CHECK (last_closed_isobar_pressure BETWEEN 800 AND 1100),
+    gust_speed INTEGER CHECK (gust_speed BETWEEN 0 AND 400),
+    sid VARCHAR NOT NULL,
+    provider VARCHAR,
+    basin VARCHAR NOT NULL,
+    nature VARCHAR,
+    valid_time TIMESTAMP NOT NULL,
+    quadrant_radius_34 TEXT NOT NULL,
+    quadrant_radius_50 TEXT NOT NULL,
+    quadrant_radius_64 TEXT,
+    point_id VARCHAR NOT NULL,
+    storm_id VARCHAR NOT NULL,
+    geometry geometry(Point,4326) NOT NULL,
+    CONSTRAINT unique_sid_storm_time UNIQUE (sid, storm_id, valid_time)
```

Comment:
For the upsert to work out of the box, I think we want to call this one "ibtracs_tracks_geo_unique". 

Should address the error in this run: https://adb-6009046713167663.3.azuredatabricks.net/jobs/638351145729392/runs/164675474861938?o=6009046713167663